### PR TITLE
utils_numeric: add function to format readable size

### PR DESCRIPTION
**Pull request by lolyu**
add function 'format_size_human_readable' to format a
number of bytesize to a human readable filesize.

Signed-off-by: lolyu <lolyu@redhat.com>

### CODE REVIEW

File: virttest/utils_numeric.py [hunk 1]
```diff
@@ -10,3 +10,30 @@ def align_value(value, factor=1024):
...
+
+    value = float(value)
+    for i, s in enumerate(suffix):
+        unit = base ** (i + 1)
+        if value < unit:
+            break
+    value = value * base / unit
+    format_str = ('%d' if value.is_integer() else format) + ' %s'
+    return format_str % (value, s)
+
```
Comment (vivianQizhu) [inline virttest/utils_numeric.py]: This should be a redundant blank line

**Comment by vivianQizhu** [comment]
Bug id please.

File: virttest/utils_numeric.py [hunk 2]
```diff
@@ -10,3 +10,29 @@ def align_value(value, factor=1024):
     :return: aligned value
     """
     return int(math.ceil(float(value) / factor) * factor)
+
+
+def format_size_human_readable(value, binary=False, format='%.2f'):
```
Comment (balamuruhans) [inline virttest/utils_numeric.py]: `format` is a python keyword, can we use different variable name `precision`.

File: virttest/utils_numeric.py [hunk 3]
```diff
@@ -10,3 +10,29 @@ def align_value(value, factor=1024):
...
+    """
+    Format a number of bytesize to a human readable filesize. By default,
+    decimal suffixes and base:10**3 will be used.
+
+    :param binary: use binary suffixes (KiB, MiB) and use 2*10 as base
+    :format: format string to specify precision for float
```
Comment (balamuruhans) [inline virttest/utils_numeric.py]: `:param param_name:`

**Comment by balamuruhans** [comment]
Thanks @lolyu @vivianQizhu, merging it.

**Comment by lolyu** [comment]
thanks @vivianQizhu @balamuruhans

File: virttest/utils_numeric.py [hunk 4]
```diff
@@ -10,3 +10,29 @@ def align_value(value, factor=1024):
...
+    """
+    Format a number of bytesize to a human readable filesize. By default,
+    decimal suffixes and base:10**3 will be used.
+
+    :param binary: use binary suffixes (KiB, MiB) and use 2*10 as base
+    :param precision: format string to specify precision for float
+    """
+
+    suffixes = {
+        'decimal': ('B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'),
```
Comment (hereischen) [inline virttest/utils_numeric.py]: 'KB',
 better add an option only use one character suffixes such as K M G

**Pull request merged**